particles/Tutorial/Kotlin/Blackjack/Player.kt

```diff
@@ -0,0 +1,39 @@
+package arcs.tutorials.blackjack
+
+import arcs.sdk.Handle
+import arcs.sdk.Utils.log
+
+class Player : AbstractPlayer() {
+    val name = "Player"
+
+    override fun getTemplate(slotName: String) = """
+        ${name}: <button type="button" on-click="onHit"> Hit </button> 
+        <button type="button" on-click="onStand"> Stand </button>
+        <span>{{hand}}</span> 
+     """.trimIndent()
+
+    override fun populateModel(slotName: String, model: Map<String, Any>): Map<String, Any> {
+        val desc = hand.joinToString(separator = ":", transform = { card ->
+            Card.cardDesc(card.value.toInt())
+        })
```

Comment:
Kotlin convention is to place the lambda outside of the closing `)` for the function call.

```kotlin
        val desc = hand.joinToString(separator = ":") { card ->
            Card.cardDesc(card.value.toInt())
        }
```

Reply:
Ah, yes... trailing closures.